documentation/src/main/asciidoc/stories/assembly_deployment_planning.adoc

```diff
@@ -9,6 +9,38 @@ To get the best performance for your {brandname} deployment, you should do the f
 * Determine what type of clustered cache mode best suits your use case and requirements.
 * Understand performance trade-offs and considerations for {brandname} capabilities that provide fault tolerance and consistency guarantees.
 
+[IMPORTANT]
+====
+{brandname} includes many configurable use cases to meet its cross-product capabilities, so determining a formula for performance metrics that covers all use cases is not possible. Additionally, consider the following inter-related factors that apply to {brandname}:
+
+* A cloud environment’s available CPU and memory resources
+* Querying capabilities
+* Entries per cache
+* Peak load and throughput limitations
+* Size of cache entries
+
+The previous listed factors can lead to an uncountable combinations of settings. Some of these settings are listed as follows:
+
+* Number of entries per cache
+* Size of entries
+* Caches used in parallel
+* Throughput
+* Peak load
+* Get, put, query balancing 
+* Power of the environment, such as memory, CPU, or the settings if running in a cloud environment.
+
+ Each setting has its own performance penalties, especially when combined with other settings. For this reason, measuring  all combinations is not possible.
+
+//Downstream content
+ifdef::downstream[]
+{RedHat} cannot publish a performance calculation that can meet all {brandname}’s use cases. The _{brandname} Deployment Planning and Sizing Guide_ details many use cases and architectures that you can analyze to determine your {brandname} configuration requirements.
+endif::downstream[]
```

Comment:
I'd remove the downstream conditional here. That statement applies to community too.